src/content/docs/apm/agents/ruby-agent/features/ruby-codestream-integration.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: New Relic CodeStream integration
+tags:
+  - Agents
+  - Ruby agent
+  - Supported features
+metaDescription: The Ruby agent can be integrated with New Relic CodeStream to provide code-level metrics.
+---
+
+Display context-sensitive APM data directly in your IDE by integrating [New Relic CodeStream](/docs/codestream/start-here/what-is-codestream) with the Ruby agent. Visualize code-level production telemetry in your editor as your write and review code with this integration.
```

Comment:
Could you update [New Relic CodeStream](/docs/codestream/start-here/what-is-codestream) to [New Relic CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level)

Reply:
Sure thing. Created #7879